### PR TITLE
Adds documentation for additional eager for procedure calls.

### PR DESCRIPTION
**Pull request by Dunfluff**

### CODE REVIEW

File: modules/ROOT/pages/extending-neo4j/procedures.adoc [hunk 1]
```diff
@@ -154,8 +154,8 @@ Particular things to note:
 *** `DBMS` -- This procedure performs system operations such as user management and query management.
     A procedure with this mode is not able to read or write graph data.
 ** `eager` is a boolean setting defaulting to `false`.
-   If it is set to `true`, the Cypher planner plans an extra `eager` operation before calling the procedure.
-   This is useful in cases where the procedure makes changes to the database in a way that could interact with the operations preceding the procedure.
+   If it is set to `true`, the Cypher planner plans an extra `eager` operation before and after calling the procedure.
+   This is useful in cases where the procedure makes changes to the database in a way that could interact with the operations preceding or proceeding the procedure.
```
Comment (AzuObs) [inline modules/ROOT/pages/extending-neo4j/procedures.adoc:158]:
```suggestion
   This is useful in cases where the procedure makes changes to the database in a way that could interact with the operations preceding or following the procedure.
```

I had a real doubt if you could use the word proceed in this way. I tried to google it and could not find a meaning of "proceed" that equalled "after".

**Review by AzuObs — APPROVED** [review]
Nice patch! Once the comment is resolved (one way or another) then feel free to merge.

File: modules/ROOT/pages/extending-neo4j/procedures.adoc (hunk 1 above)
Comment (Dunfluff) [reply modules/ROOT/pages/extending-neo4j/procedures.adoc:158]: Ah, i got the wrong latin word it should be postceding but that just sounds wrong so let's go with following.

**Review by renetapopova — APPROVED** [review]
Looks good! Thanks @Dunfluff!

**Pull request merged**